i18n(it): update why-astro.mdx (#6621)
withastro/docs: src/content/docs/it/concepts/why-astro.mdx

---
title: Perché Astro?
description: "Astro è il framework web per la creazione di siti web basati su contenuti come blog, marketing ed e-commerce. Scopri perché Astro potrebbe essere una buona scelta per il tuo prossimo sito web."
i18nReady: true
---

**Astro** è il framework web per la creazione di **siti web basati su contenuti** come blog, marketing ed e-commerce. Astro è meglio conosciuto per aver introdotto una nuova [architettura frontend](/it/concepts/islands/) per ridurre l'overhead e la complessità del JavaScript rispetto ad altri framework. Se hai bisogno di un sito web che carichi velocemente e abbia un'ottima SEO, allora Astro fa al caso tuo.

## Caratteristiche

> **Astro è un framework web all-in-one**. Include tutto ciò di cui hai bisogno per creare un sito web, già integrato. Ci sono anche centinaia di diverse [integrazioni](https://astro.build/integrations/) e [hook API](/in/reference/integrations-reference/) disponibili per personalizzare un progetto in base alle tue esigenze e al tuo caso d'uso specifico.

Reviewer: ```suggestion
**Astro è un framework web all-in-one**. Include tutto ciò di cui hai bisogno per creare un sito web, già integrato. Ci sono anche centinaia di diverse [integrazioni](https://astro.build/integrations/) e [hook API](/it/reference/integrations-reference/) disponibili per personalizzare un progetto in base alle tue esigenze e al tuo caso d'uso specifico.
```

Alcune caratteristiche principali includono:

- **[Isole](/it/concepts/islands/):** Un'architettura web basata su componenti ottimizzata per siti web incentrati sui contenuti.

> - **[UI-agnostic](/it/core-concepts/framework-components/):** Supporta React, Preact, Svelte, Vue, Solid, Lit, HTMX, componenti web e altro ancora.

Reviewer: ```suggestion
- **[UI-agnostic](/it/guides/framework-components/):** Supporta React, Preact, Svelte, Vue, Solid, Lit, HTMX, componenti web e altro ancora.
```

> - **[Server-first](/it/core-concepts/rendering-modes/):** Sposta il rendering dispendioso fuori dai dispositivi dei visitatori.

Reviewer: ```suggestion
- **[Server-first](/it/basics/rendering-modes/):** Sposta il rendering dispendioso fuori dai dispositivi dei visitatori.
```

> - **[Zero JS, di default](/it/core-concepts/astro-components/):** Meno JavaScript lato client per rallentare il tuo sito.

Reviewer: ```suggestion
- **[Zero JS, di default](/it/basics/astro-components/):** Meno JavaScript lato client per rallentare il tuo sito.
```

### Orientato al contenuto

**Astro è stato progettato per la creazione di siti web ricchi di contenuti.** Ciò include la maggior parte dei siti di marketing, siti di pubblicazione, siti di documentazione, blog, portfolio e alcuni siti di e-commerce.

Al contrario, la maggior parte dei moderni web framework sono progettati per la creazione di *applicazioni web*. Questi framework funzionano meglio per la creazione di esperienze più complesse simili a quelle delle applicazioni nel browser: dashboard di amministrazione con accesso, caselle di posta, social networks, liste di attività e persino applicazioni simili a quelle native come [Figma](https://figma.com/) e [Ping](https://ping.gg/).

L'attenzione di Astro sui contenuti fin dalle sue origini come costruttore di siti statici ha permesso ad Astro di **scalare in modo sensato verso applicazioni web dinamiche, potenti e performanti** che rispettano ancora i tuoi contenuti e il tuo pubblico. L'unico focus di Astro sui contenuti permette ad Astro di fare scelte e fornire caratteristiche di performance ineguagliate che non avrebbero senso per altri framework web più focalizzati sulle applicazioni.

### Server-first

> **Astro sfrutta il [rendering lato server](/it/core-concepts/rendering-modes/) il più possibile rispetto al rendering lato client.** Questo è lo stesso approccio che i tradizionali framework lato server come PHP, WordPress, Laravel, Ruby on Rails, ecc. hanno utilizzato per decenni. Ma non è necessario imparare un secondo linguaggio lato server per sfruttarlo. Con Astro, tutto è ancora solo HTML, CSS e JavaScript (o TypeScript, se preferisci).

Reviewer: ```suggestion
**Astro sfrutta il [rendering lato server](/it/basics/rendering-modes/) il più possibile rispetto al rendering lato client.** Questo è lo stesso approccio che i tradizionali framework lato server come PHP, WordPress, Laravel, Ruby on Rails, ecc. hanno utilizzato per decenni. Ma non è necessario imparare un secondo linguaggio lato server per sfruttarlo. Con Astro, tutto è ancora solo HTML, CSS e JavaScript (o TypeScript, se preferisci).
```
